### PR TITLE
fix(integrations): fixes SENTRY-11V7

### PR DESCRIPTION
**Pull request by scefali**
Stop sending integration not found errors to Sentry

### CODE REVIEW

File: src/sentry/integrations/github/webhook.py [hunk 1]
```diff
@@ -76,7 +76,6 @@ def __call__(self, event: Mapping[str, Any], host: str | None = None) -> None:
                     "external_id": str(external_id),
                 },
             )
-            logger.exception("Integration does not exist.")
```
Comment (ceorourke) [inline src/sentry/integrations/github/webhook.py:79]: I think @armenzg added this for some specific reason, would be good to ask him.

Comment (armenzg) [reply src/sentry/integrations/github/webhook.py:79]: Let's turn it to a warning.

Ideally, we would investigate *why* we got into this state and prevent it from happening.
Or even tracking it as an analytic so we can see if something gets worse or fixed.

File: src/sentry/integrations/github_enterprise/webhook.py [hunk 2]
```diff
@@ -32,12 +32,13 @@ def get_installation_metadata(event, host):
...
 
+    external_id = "{}:{}".format(host, event["installation"]["id"])
     integration = integration_service.get_integration(
-        external_id="{}:{}".format(host, event["installation"]["id"]),
+        external_id=external_id,
         provider="github_enterprise",
     )
     if integration is None:
-        logger.exception("Integration does not exist.")
+        logger.info("Integration does not exist.", extra={"external_id": external_id})
```
Comment (armenzg) [inline src/sentry/integrations/github_enterprise/webhook.py:41]:
```suggestion
        logger.warning("Integration does not exist.", extra={"external_id": external_id})
```

File: src/sentry/integrations/github/webhook.py (hunk 1 above)
Comment (scefali) [reply src/sentry/integrations/github/webhook.py:79]: @armenzg pretty sure this happens if you just uninstall the Github integration in Sentry. We already log the message at the info level above. If it's warning level, it's going to go to Sentry but info level just goes to logging.

Comment (armenzg) [reply src/sentry/integrations/github/webhook.py:79]: @scefali error, critical and exception goes to Sentry. warning does not.

**Pull request closed without merging**